modules/userId/index.js

```diff
@@ -727,6 +781,7 @@ export function init(config) {
   (getGlobal()).getUserIds = getUserIds;
   (getGlobal()).getUserIdsAsEids = getUserIdsAsEids;
   (getGlobal()).refreshUserIds = refreshUserIds;
+  (getGlobal()).findRootDomain = findRootDomain;
```

Comment:
The plan for this function is to be used within pbjs? So, it won't be executed outside pbjs?  
If that is the case then maybe we should not make this function accessible on `pbjs` namespace, but rather just `exporting` the function and then just using it inside some module?

If you decide to go that way (exporting instead of marking it available on `pbjs`) I suggest you move it in `utils` package.

Reply:
Good idea, I have done so

Reply:
Sorry for going back and forth, I didn't think this through, my bad. Now I see that `utils` may not be the best place to put this function.  
Now  `utils` has dependency to `storageManager.js`, which should not happen. `utils` should not depend on `storageManager`.  
Can you move this function to `userId/index.js` (again) and just leave it as `exported` instead of attaching it on `pbjs` namespace?

Reply:
Hmm, yeah, I was on the fence about that so that's why I put it in the user module initially but now I'm not sure how my module will get access to it. The linter forbids me from importing directly from userId/index.js. Do you have any thoughts on that? What do you think about adding the root domain to the config object that gets passed into `getId`

Reply:
Or maybe in userSync.js?

Reply:
I would prefer `userSync.js`.  
@smenzer do you have any opinion on this one?

Reply:
I don't think we should add it to the config that gets passed to `getId`, because that is specifically the config for that module. we could add it as a new parameter, though. would need to get passed to `extendId` as well.

the only issue i have with moving it to `userSync.js` is that we aren't even using that module today within userId module, so seems odd to put it there just because it's somewhat related.

since we can't use `utils`, which is where I would have expected to put this (but I understand the reason not to), and since putting it in `userId/index.js` won't work for using it within submodules...i think we're really left with two options:

1. put it back on the global `pbjs` namespace
2. pass it in the function calls to the modules

if we pass the result of the function to all modules, then it means we're executing that code EVERY time, which isn't necessary since most modules aren't using it today. we could pass it as a function to be used if necessary, but not sure if that's a good idea or not. which leads us back to putting it in the `pbjs` namespace even though it may not be used outside of `pbjs` ... 

not a real answer i know, but that's how i'm thinking about it. maybe we can get some other opinions in the slack group, too?

Reply:
Checking in to see if there is consensus on an approach that I should take, thanks!